src/main/java/org/opensearch/ad/transport/AnomalyResultTransportAction.java

```diff
@@ -812,7 +837,7 @@ private boolean invalidQuery(SearchPhaseExecutionException ex) {
         @Override
         public void onResponse(RCFResultResponse response) {
             try {
-                stateManager.resetBackpressureCounter(rcfNodeID);
+                stateManager.resetBackpressureCounter(rcfNodeID, adID);
```

Comment:
Why do we reset here?

Reply:
A successful responses means the node is responsive and working.  Will add a comment.